zombienet_tests/README.md

```diff
@@ -12,6 +12,49 @@ _The content of this directory is meant to be used by Parity's private CI/CD inf
 * [zombienet repo](https://github.com/paritytech/zombienet)
 * [zombienet book](https://paritytech.github.io/zombienet/)
 
+## Running tests locally
+
+To run any test locally use the native provider (`zombienet test -p native ...`) you need first build the binaries. They are:
+
+* adder-collator -> polkadot/target/testnet/adder-collator
+* malus -> polkadot/target/testnet/malus
+* polkadot -> polkadot/target/testnet/polkadot
+* polkadot-collator -> cumulus/target/release/polkadot-parachain
+* undying-collator -> polkadot/target/testnet/undying-collator
+
+To build them use:
+* adder-collator -> `cargo build --profile testnet -p test-parachain-adder-collator`
+* undying-collator -> `cargo build --profile testnet -p test-parachain-undying-collator`
+* malus -> cargo build --profile testnet -p polkadot-test-malus
+* polkadot (in polkadot repo) and polkadot-collator (in cumulus repo) -> `cargo build --profile testnet`
+
+One solution is to use the `.set_env` file (from this directory) and fill the `CUSTOM_PATHS` before *source* it to patch the PATH of your system to find the binaries you just built.
+
+E.g.:
+```
+$ cat .set_env
+(...)
+# by the order of this array
+CUSTOM_PATHS=(
+  "<path>/polkadot/target/release"
+  "<path>/polkadot/target/testnet"
+  "<path>/cumulus/target/release"
+)
+(...)
+
+source .set_env
+```
+
+Then you have your `PATH` customized and ready to run `zombienet`.
+ **NOTE**: You should need to do this ones per terminal session, since we are patching the `PATH` and re-exporting.
```

Comment:
You can also source this env fine in `.bashrc` file for example which gets executed when you log in. So you don't have to do this every time.

Reply:
Yes, adding to `.bashrc` will be better and save some time.
Thanks!

Reply:
We go in the personal preference zone :)
One might want to source it by hand, other - in bashrc, third (me :P ) - alias it and so on.

We've got a hint to use path, we've got a script to avoid useless copy-pasting/linking, we even have got sane-ish defaults. Let's merge as it is.